packages/salesforcedx-apex-debugger/test/unit/adapter/apexDebug.test.ts

```diff
@@ -206,6 +211,10 @@ describe('Interactive debugger adapter - unit', () => {
       }
     });
 
+    afterAll(() => {
+      adapter.clearIdleTimers();
```

Comment:
Interesting that `adapter.clearIdleTimers()` now needs to be called

Reply:
Yeah, basically jest reports left over open handle (pending promises and timeouts) where mocha was just ignoring them.